mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = async (req, res) => {
+  try {
+    const deletionResult = await Order.findOneAndDelete({ _id: req.params.orderId });
+    if (!deletionResult) {
+      return res.status(404).json({ error: "Order not found" });
+    }
+    res.json({ message: "Order removed successfully" });
+  } catch (error) {
+    res.status(500).json({ error: errorHandler(error) });
```

Comment:
In the `catch` block, it's better to log the error before sending the response. This will help in debugging and tracking errors. Add a log statement before `res.status(500).json({ error: errorHandler(error) });`.